fw/http_limits.c

```diff
@@ -1467,6 +1468,13 @@ frang_tls_handler(TlsCtx *tls, int state)
 	TfwVhost *dflt_vh = tfw_vhost_lookup_default();
 	int r;
 
+	switch (state) {
+	case TTLS_HS_CB_FINISHED_NEW:
+	case TTLS_HS_CB_FINISHED_RESUMED:
+		TFW_INC_STAT_BH(serv.tls_hs_successful);
+		break;
+	}
```

Comment:
It looks correct, but I suggest to move this stat incrementation to `T_FSM_STATE(TTLS_HANDSHAKE_OVER)` in tls_srv.c

Reply:
From tls/tls_srv.c we haven't access to fw/procfs.h, because it's separate library, as I understood.

Reply:
I can suggest to add separated callback for this, but it's up to you.

Reply:
I understand this code smell, but it seems acceptable for me. Any option will suit me.

Reply:
Added separate callback.